### PR TITLE
Add validation scripts for the Vortex shedding example

### PR DESCRIPTION
**Pull request by ktangsali**
# Modulus Pull Request

## Description
Adds a physics orientated model inference and metrics. 

## Checklist

- [ ] I am familiar with the [Contributing Guidelines](https://github.com/NVIDIA/modulus/blob/main/CONTRIBUTING.md).
- [ ] New or existing tests cover these changes.
- [ ] The documentation is up to date with these changes.
- [ ] The [CHANGELOG.md](https://github.com/NVIDIA/modulus/blob/main/CHANGELOG.md) is up to date with these changes.
- [ ] An [issue](https://github.com/NVIDIA/modulus/issues) is linked to this pull request.

### CODE REVIEW

File: examples/cfd/vortex_shedding_mgn/inference.py [hunk 1]
```diff
@@ -84,7 +84,7 @@ def __init__(self, cfg: DictConfig, logger: PythonLogger):
             models=self.model,
             device=self.device,
         )
-
+
```
Comment (Alexey-Kamenev) [inline examples/cfd/vortex_shedding_mgn/inference.py:87]: Redundant spaces (probably won't pass CI as-is).

File: examples/cfd/vortex_shedding_mgn/inference.py [hunk 2]
```diff
@@ -156,6 +156,12 @@ def predict(self):
             self.faces.append(torch.squeeze(cells).numpy())
             self.graphs.append(graph.cpu())
 
+    def get_raw_data(self, idx):
+        self.pred_i = [var[:, idx] for var in self.pred]
+        self.exact_i = [var[:, idx] for var in self.exact]
+
+        return(self.graphs, self.faces, self.pred_i, self.exact_i)
```
Comment (Alexey-Kamenev) [inline examples/cfd/vortex_shedding_mgn/inference.py:163]: Add space and parentheses are not needed.

File: examples/cfd/vortex_shedding_mgn/inference_analysis/utils.py [hunk 3]
```diff
@@ -0,0 +1,217 @@
+import numpy as np
+import modulus.sym
```
Comment (Alexey-Kamenev) [inline examples/cfd/vortex_shedding_mgn/inference_analysis/utils.py:2]: I don't think it's used in this file.

**Review by Alexey-Kamenev — APPROVED** [review]
LGTM but please run `pre-commit`.

File: examples/cfd/vortex_shedding_mgn/inference.py (hunk 1 above)
Comment (ktangsali) [reply examples/cfd/vortex_shedding_mgn/inference.py]: Thanks! Fixed it - turns out the `black` installation in my env was outdated.

File: examples/cfd/vortex_shedding_mgn/inference.py (hunk 2 above)
Comment (ktangsali) [reply examples/cfd/vortex_shedding_mgn/inference.py]: Fixed!

File: examples/cfd/vortex_shedding_mgn/inference_analysis/utils.py (hunk 3 above)
Comment (ktangsali) [reply examples/cfd/vortex_shedding_mgn/inference_analysis/utils.py]: Good catch!

**Comment by mnabian** [comment]
LGTM, thanks!

**Review by mnabian — APPROVED** [review]

**Pull request merged**